app/views/posts/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<%= form_with model: post, url: user_posts_path, method: :post, local: true, scope: :add_post do |form| %>
+
+  <div>
+    <%= form.label :title %><br>
+    <%= form.text_field :title %>
+  </div>
+
+  <div>
+    <%= form.label :text %><br>
+    <%= form.text_area :text, size: "80x6" %>
+  </div>
+
+  <%= form.submit %>
+
+  <% end %>
```

Comment:
- [ ] Kindly add a form for the comments to allow a user to add comments. This should be done under the views of the comments.

Reply:
Please I have a form for adding comments. Please, my `CommentsController` does not need a `#new` action because the page that displays the comments for a particular post has a form for creating a new comment so I believe having a view for the `new` is unnecessary. Please kindly consider how it is on a social media site, Most `comment boxes` are found beneath that particular post and not an entirely new post.

Please consider this 🙏 

Thank you.